tests/Tokenio/MemberRegistrationTest.php

```diff
@@ -137,4 +141,44 @@ public function testRecovery()
         $this->assertEquals([$alias], $actualAliases);
     }
 
+    public function testRecoveryWithSecondaryAgent()
+    {
+        $this->setUp();
```

Comment:
setup is automatically called right?

Reply:
That was added for testing purpose, I'll remove that